pgdb/v1/fts.go

```diff
@@ -108,6 +106,53 @@ func FullTextSearchVectors(docs []*SearchContent, additionalFilters ...jargon.Fi
 			_ = err
 		}
 	}
+	return rv
+}
+
+func camelSplitDocs(docs []*SearchContent) []lexeme {
+	rv := make([]lexeme, 0, 8)
+	camelCaseRegex := regexp.MustCompile(`[[:upper:]][[:lower:]]+`)
```

Comment:
this should at least be module scoped

Reply:
Yup, done in 4d22e33be3f06e9da0ead3c523e26131d1ca2695